scheduler-server/sql/scheduler/tables/schema_migrations.sql

```diff
@@ -0,0 +1,21 @@
+create table "schema_migrations" (
+  "version" varchar not null primary key
```

Comment:
Additional thought: AFAIK moving things out of the public schema is something that's on our roadmap, so this should probably be moved this to it's own "migrations" schema or something of the like, so that it doesn't have to be migrated out of there later.